tests/submodule/lookup.c

```diff
@@ -42,6 +42,46 @@ void test_submodule_lookup__simple_lookup(void)
 	assert_submodule_exists(g_repo, "sm_added_and_uncommited/");
 }
 
+void test_submodule_lookup__can_be_dupped(void)
+{
+	git_submodule *sm;
+	git_submodule *sm_duplicate;
+	const char *oid = "480095882d281ed676fe5b863569520e54a7d5c0";
+
+	// Check original
```

Comment:
We don't use `//` comments in this project...

```suggestion
	/* Check original */
```

Reply:
@ethomson 
Fixed